src/ckb-daemon/usb_linux.c

```diff
@@ -561,7 +565,6 @@ int os_setupusb(usbdevice* kb) {
     ///
     const char* ep_str = udev_device_get_sysattr_value(dev, "bNumInterfaces");
 #ifdef DEBUG
-    ckb_info("Connecting %s at %s%d\n", kb->name, devpath, index);
     ckb_info("claiming interfaces. name=%s, serial=%s, firmware=%s; Got >>%s<< as ep_str\n", name, serial, firmware, ep_str);
```

Comment:
Not sure whether we need to print a serial code. Since we are concerned about security and even sign the FIRMWARE, I guess it shouldn't be printed in the logs that people publish here afterwards not realizing they are publishing it as it might be misused.